cmd/run.go

```diff
@@ -32,7 +32,7 @@ import (
 // NewCmdRun creates a new run command
 func NewCmdRun() *cobra.Command {
 	flagMapping := config.RunFlagsNameMapping{
-		ApiListeningAddress:  "apiListeningAddress",
+		ApiListeningAddress:  "apiAddress",
```

Comment:
is there another ApiAddress name? Then let's name the struct's attribute `ApiAddress` as well, to have harmonic naming.